addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -77,7 +77,7 @@ function factory(dependencies) {
              * connection that were established but failed or timed out.
              */
             browser.setInterval(() => {
-                this.channel_id && this.currentRtcSession && this._callSessions();
+                this.channelId && this.currentRtcSession && this._callSessions();
```

Comment:
Isn't the correct fix `this.channel` instead?

Reply:
Also the `setInterval` should be killed in `_willDelete`.

Reply:
changes done